src/plugins/DevTools/components/DevToolsComponent.ts

```diff
@@ -17,6 +18,15 @@ export class DevToolsComponent extends OwlUIComponent<{}, DevToolsState> {
         closed: true,
         currentTab: 'inspector',
         height: 300,
+        actions: [], // Stack of all actions performed since init
+    };
+    handlers: PluginHandlers = {
+        intents: {
+            '*': 'addAction',
+        },
+    };
+    commands = {
+        addAction: this.addAction.bind(this),
```

Comment:
I'm thinking about something. You remember when we talked about namespacing the commands ? What about something like `DevTools:addAction` ? Then you could hook anything in the same way. You want to hook an "intent" (or a "command" if we rename it), you hook `addAction`. You want to hook the specific addAction command of the DevTools ? You hook `DevTools:addAction`. You want to hook the plugin Whatever which is already hooking `DevTools:addAction` ? You hook `Whatever:addAction`. You want to hook the `wrap` primitive ? You hook `Document:wrap`. That's it. One mechanism. (Just an idea for later.)

Reply:
I like that idea very much. To explore!